tests/config/config_parse_test.py

```diff
@@ -1157,18 +1131,8 @@ def test_validator_passes_with_context(self):
         )
         assert self.validator(template, context)
 
-    def test_validator_valid_format_string(self):
-        template = "The {one} thing I {seven} is {mars}"
-        context = config_utils.ConfigContext(
-            path=None,
-            nodes=None,
-            command_context={'mars': 'ok'},
-            namespace=None,
-        )
-        assert self.validator(template, context)
-
     def test_validator_valid_percent_string(self):
```

Comment:
do we still need this test? if we do, let's rename

Reply:
I've rewritten the test to see if we don't need a %% escape anymore.